test/object-store/util/sync/baas_admin_api.hpp

```diff
@@ -282,6 +283,9 @@ std::string get_real_base_url();
 // your test is talking to.
 std::string get_admin_url();
 
+// Returns a reference to the redirector if it is enabled.
+std::optional<sync::RedirectingHttpServer>& get_redirector();
```

Comment:
Instead of pulling the redirect server in as an include dependency of anyone using the baas admin API why not just instantiate one where you need it?

Reply:
Removed the function in `baas_admin_api.hpp` and the test is now declaring its own local redirect server.